2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py

```diff
@@ -0,0 +1,21 @@
+# made by vanguard7
+def main():
+    t = int(input())
+    for i in range(t):
+        n = int(input())
+        low = 10000000000000000000000000000000000000000000
```

Comment:
Just asking out of curiosity, How many zeroes are these?

Reply:
@aashutoshrathi: Didn't count! 😄